api/envoy/api/v2/cluster/circuit_breaker.proto

```diff
@@ -5,8 +5,6 @@ package envoy.api.v2.cluster;
 option java_outer_classname = "CircuitBreakerProto";
 option java_multiple_files = true;
 option java_package = "io.envoyproxy.envoy.api.v2.cluster";
-option csharp_namespace = "Envoy.Api.V2.ClusterNS";
```

Comment:
I'm pretty sure someone added this at some point to fix something? Can we remove them? Same elsewhere.

Reply:
I think I'm going to have to auto-generate this to avoid the issue in https://github.com/envoyproxy/envoy/pull/3854. Will make this fix in this PR.

Reply:
I've restored these as a special case workaround for v2. In v3, we will fix this structurally (already tracked at https://github.com/envoyproxy/envoy/issues/8120).